.travis.yml

```diff
@@ -1,3 +1,24 @@
 language: go
+sudo: false
+
+env:
+  - GO111MODULE=on
+
+git:
+  depth: 1
+
+go:
+  - 1.12.x
+  - 1.13.x
+  - tip
+
+before_script:
+  - go get github.com/mattn/goveralls
+
 script:
-  - env GO111MODULE=on go test ./... -short
+  - go test -v -race -short -coverprofile=coverage.out ./...
```

Comment:
Tests can actually run by `goveralls`, but `-short` option is not currently supported so I used plain `go test` at this point. In my opinion, it would be nice if we can remove `-short` from .travis.yml so all integration tests including `consul` can be tested on CI. This could be addressed in another p-r or issue, I suppose.